scripts/index.js

```diff
@@ -0,0 +1,148 @@
+const tempState = {
+    temp: 70
+};
+
+const temperature = document.querySelector("#temperature");
+const landscape = document.querySelector("#landscapeContainer");
+
+const hotter = (event) => {
+    tempState.temp +=1;
+    temperature.textContent = `${tempState.temp}F`;
+    temperature.style.color = tempColor();
+    checkLand();
+    landText();
+};
+
+const colder = (event) => {
+    tempState.temp -=1;
+    // const landscape = document.querySelector("#landscapeContainer");
+    temperature.textContent = `${tempState.temp}F`;
+    temperature.style.color = tempColor();
+    checkLand();
+    landText();
+};
```

Comment:
A couple of notes:

1.  You're not using the `event` parameter so you can drop it.
2. You can combine these functions into one.

```suggestion
const changeTemp = (increment) => {
    tempState.temp += increment;
    temperature.textContent = `${tempState.temp}F`;
    temperature.style.color = tempColor();
    checkLand();
    landText();
};
```

Then you can use an anonymous function to call it when you register the event handlers.

```javascript
    colderButton.addEventListener("click", () => changeTemp( -1 ) );
```